Sources/Instrumentation/Instrument.swift

```diff
@@ -51,22 +51,22 @@ public protocol Injector: _SwiftInstrumentationSendable {
 /// Conforming types are usually cross-cutting tools like tracers. They are agnostic of what specific `Carrier` is used
 /// to propagate metadata across boundaries, but instead just specify what values to use for which keys.
 public protocol Instrument: _SwiftInstrumentationSendable {
-    /// Extract values from a `Carrier` by using the given extractor and inject them into the given `Baggage`.
+    /// Extract values from a `Carrier` by using the given extractor and inject them into the given `ServiceContext`.
     /// It's quite common for `Instrument`s to come up with new values if they weren't passed along in the given `Carrier`.
     ///
     /// - Parameters:
     ///   - carrier: The `Carrier` that was used to propagate values across boundaries.
-    ///   - baggage: The `Baggage` into which these values should be injected.
+    ///   - baggage: The `ServiceContext` into which these values should be injected.
     ///   - extractor: The ``Extractor`` that extracts values from the given `Carrier`.
-    func extract<Carrier, Extract>(_ carrier: Carrier, into baggage: inout Baggage, using extractor: Extract)
+    func extract<Carrier, Extract>(_ carrier: Carrier, into baggage: inout ServiceContext, using extractor: Extract)
```

Comment:
```suggestion
    func extract<Carrier, Extract>(_ carrier: Carrier, into serviceContext: inout ServiceContext, using extractor: Extract)
```
I think mixing the wording might cause a bit of confusion, so perhaps we should go all in with `serviceContext`?

Reply:
It seems we already picked `context`?
```suggestion
    func extract<Carrier, Extract>(_ carrier: Carrier, into context: inout ServiceContext, using extractor: Extract)
```